src/components/paginationButton/paginationButton.js

```diff
@@ -4,7 +4,6 @@ export default function PaginationButton({bookTitle, sort, category, loadMore, v
     const [startIndex, setStartIndex] = useState(30);
     const handleClick = () => {
         let maxResults = (resultsValue - startIndex) < 30 ? (resultsValue - startIndex) : 30;
```

Comment:
30 is a common value, you should set the constant

Reply:
Ok